### PR TITLE
fix dropdowns have wrong mark color in browsers

### PR DESCRIPTION
**Pull request by Darshan-upadhyay1110**
* Resolves: #3235
* Target version: main

### Summary
- `cool-var-source-light` element is no longer loaded to office.vue
- create element div to handle null value foe light/dark element

### TODO

- [ ] ...

### Checklist

- [x] Code is properly formatted
- [x] Sign-off message is added to all commits
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by Darshan-upadhyay1110** [comment]
CC: @juliushaertl can you please review this patch.

File: src/helpers/coolParameters.js [hunk 1]
```diff
@@ -88,8 +95,7 @@ const generateCSSVarTokens = () => {
 	}
 
 	// New dark mode compatible way to hand over our Nextcloud variables in both light/dark to Collabora
-	const lightElement = document.getElementById('cool-var-source-light') ?? document.documentElement
-	const darkElement = document.getElementById('cool-var-source-dark') ?? document.documentElement
+	const darkElement = createDataThemeDiv('div', 'dark')
```
Comment (juliushaertl) [inline src/helpers/coolParameters.js:98]: Can we also remove those elements after extracting the values again?

**Review by juliushaertl — COMMENTED** [review]
Small nitpick to cleanup the elements, otherwise tested and works 👍

**Comment by juliushaertl** [comment]
Also eslint is not happy:

> Error:   54:6  error  'element' is never reassigned. Use 'const' instead  prefer-const

**Comment by Darshan-upadhyay1110** [comment]
- Data theme element removed after value extraction.
- lint issue is being addressed.
- Backport PR: https://github.com/nextcloud/richdocuments/pull/3242

CC: @juliushaertl

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**